streaming/src/main/scala/org/apache/spark/streaming/receiver/ActorReceiver.scala

```diff
@@ -149,43 +181,61 @@ private[streaming] class ActorReceiver[T: ClassTag](
   class Supervisor extends Actor {
 
     override val supervisorStrategy = receiverSupervisorStrategy
-    val worker = context.actorOf(props, name)
-    logInfo("Started receiver worker at:" + worker.path)
-
-    val n: AtomicInteger = new AtomicInteger(0)
-    val hiccups: AtomicInteger = new AtomicInteger(0)
-
```

Comment:
Correct, volatile is not necessary.  https://groups.google.com/forum/#!msg/scalaz/kFnICLFjO-4/GT_59mZLrFAJ